thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/utils/TriggerEventListenerConfiguration.java

```diff
@@ -0,0 +1,97 @@
+package org.apache.pinot.thirdeye.anomaly.detection.trigger.utils;
+
+import java.util.List;
+
+
+public class TriggerEventListenerConfiguration {
+  private String consumerClass;
+  private String kafkaBootstrapServers;
+  private String kafkaTopic;
+  private String kafkaConsumerGroupId;
+  private String kafkaConsumerPropPath;
+  private int parallelConsumerNum = 1; // run one consumer by default
```

Comment:
Let's put them in the configuration file.

Reply:
+1

Reply:
This value can be configured by `detector.yml`.